helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ParticipantMessageMonitorMBean.java

```diff
@@ -21,7 +20,7 @@
 
 import org.apache.helix.monitoring.SensorNameProvider;
 
-
+@Deprecated
```

Comment:
I think you can juat remove it, any other usages?

Reply:
Would like to keep a record. No need? No other usage at all.

Reply:
Let's remove it. This is 100% dead code. Leaving it in the repo will cause much confusing.